src/components/partnership/partnership.service.ts

```diff
@@ -578,27 +578,28 @@ export class PartnershipService {
         -[:baseNode { active: true }]->(partnership)-[:agreementStatus { active: true }]->(agreementStatus:Property { active: true })
         RETURN total, partnership.id as id, agreementStatus.value as agreementStatus, partnership.createdAt as createdAt
         ORDER BY ${sort} ${order}
-        SKIP $skip LIMIT $count
       `;
 
-      const projectPartners = await this.db
+      let projectPartners = await this.db
         .query()
         .raw(query, {
           token: session.token,
           requestingUserId: session.userId,
           owningOrgId: session.owningOrgId,
           projectId,
-          skip: (page - 1) * count,
-          count,
         })
         .run();
 
+      result.total = projectPartners.length;
+      result.hasMore = count * page < result.total ?? true;
+
+      projectPartners = projectPartners.splice((page - 1) * count, count);
```

Comment:
We definitely want to keep the db determining the current page, otherwise it has to send all the results just for us to only use some of them.